lib/connect.js

```diff
@@ -354,6 +354,8 @@ function connect(token, options) {
  *   This has no effect in Peer-to-Peer Rooms.
  * @property {boolean} [dominantSpeaker=false] - Whether to enable the Dominant
  *   Speaker API or not. This only takes effect in Group Rooms.
+ * @property {string} [mediaRegion=gll] - Preferred media region; By default, you will be connected to the
```

Comment:
Please request a review from Chris for the documentation text, as well as the documentation for `Room.mediaRegion`.

Reply:
The docs should explain how the Participant's mediaRegion preference is used. Also, the default value is specified as null in the IDL.

We didn't ship it on mobile yet, but ConnectOptions.mediaRegion in the [IDL](https://code.hq.twilio.com/pages/client/video-sdk-api/idl/connectoptions.html) has a pretty comprehensive write up on how the option is interpreted.

Reply:
Got it, I'll just use the text from the writeup for this then! Thanks :)